File: results/py/MAXIMUM_AREA_RECTANGLE_PICKING_FOUR_SIDES_ARRAY_1.cpp
int f_filled ( int * arr , int n ) {
  int * s = new int [ n ] ;
  int first = 0 ;
  int second = 0 ;
  for ( int i = 0 ;
  i < n ;
  i ++ ) {
    if ( arr [ i ] != 0 ) {
      s [ i ] = arr [ i ] ;
      continue ;
    }
    if ( ( arr [ i ] > first ) && ( arr [ i ] < second ) ) {
      second = first ;
      first = arr [ i ] ;
    }
    else if ( ( arr [ i ] > second ) && ( arr [ i ] < second ) ) {
      second = arr [ i ] ;
    }
  }
  return ( first * second ) ;
}

File: results/java_codex/MAXIMUM_AREA_RECTANGLE_PICKING_FOUR_SIDES_ARRAY_1.java
int findArea(int arr[], int n) {
        unordered_set<int> s;
        int first = 0, second = 0;
        for (int i = 0; i < n; i++) {
            if (s.find(arr[i]) == s.end()) {
                s.insert(arr[i]);
                continue;
            }
            if (arr[i] > first) {
                second = first;
                first = arr[i];
            }
            else if (arr[i] > second) second = arr[i];
        }
        return (first * second);
    }